Postman/EnvisoApiAuthentication.postman_collection.json

```diff
@@ -428,10 +428,18 @@
 					"    pm.sendRequest(getTokenRequest, (err, response) => {",
 					"        //console.log(response);",
 					"        const jsonResponse = response.json();",
+					"        console.log(jsonResponse);",
+					"",
+					"        let status = response.status;",
+					"        console.log(\"Login Request Statuscode:\", status);",
 					"",
 					"        //console.log(\"authtoken\"+ jsonResponse.authToken)",
 					"        pm.environment.set('access_token', jsonResponse.authToken);",
 					"",
+					"        if (status !== \"OK\") {",
+					"            console.log(\"Login was not successfull, reset tokenExpireDate to try login again.\")",
+					"            pm.collectionVariables.set(\"tokenExpireDate\", 1688401934573);",
```

Comment:
This ensures the tokenExpireDate is always older than 1h so the login will be triggered again.